events_test.go

```diff
@@ -37,22 +37,15 @@ import (
 func TestRoundtrip(t *testing.T) {
 	for _, tc := range eventRoundTripTestTable {
 		t.Run(tc.Filename, func(t *testing.T) {
-			f, err := os.Open(tc.Filename)
-			if err != nil {
-				t.Fatal(err.Error())
-			}
-			defer f.Close()
-			input, err := io.ReadAll(f)
-			if err != nil {
-				t.Fatal(err.Error())
-			}
+			input, err := os.ReadFile(tc.Filename)
+			require.NoError(t, err)
 
 			event, err := UnmarshalAny(input)
-			assert.NoError(t, err)
+			require.NoError(t, err)
```

Comment:
Nit: not really relevant to the change, but I guess it is fine :)

Reply:
I know. I couldn't cope with the inconsistency of using require for the ReadFile call but not for UnmarshalAny.